sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileIO.java

```diff
@@ -426,6 +426,28 @@ public String readFullyAsUTF8String() throws IOException {
     public String toString() {
       return "ReadableFile{metadata=" + metadata + ", compression=" + compression + '}';
     }
+
+    @Override public boolean equals(Object o) {
```

Comment:
format the changed lines with `google-auto-format`?

Reply:
It's actually valid. Weird